app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt

```diff
@@ -90,11 +93,32 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
             offerToolbar.setNavigationOnClickListener { onBackPressed() }
             offerToolbar.setOnMenuItemClickListener(::handleMenuItem)
 
+            val openQuoteDetails = { quoteID: String ->
+                lifecycleScope.launch {
+                    val quoteDetailItems = model.getQuoteDetailItems(quoteID) ?: return@launch
```

Comment:
So instead of handling the result here, we update the view with a "open details activity"-event.